cpp-ch/local-engine/Storages/SubstraitSource/ParquetFormatFile.h

```diff
@@ -7,6 +7,17 @@
 #    include <IO/ReadBuffer.h>
 #    include <Storages/SubstraitSource/FormatFile.h>
 
+// clang-format off
```

Comment:
No need to set clang-format off for includes. Make sure clang-format formats all codes.